src/main/java/core/basesyntax/service/DataParser.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface DataParser {
+    int ACTIVITY_INDEX = 0;
+    int FRUIT_INDEX = 1;
+    int NUMBER_INDEX = 2;
+
+    static FruitTransaction getData(String data) {
```

Comment:
what about access modifier? It's better to awoid static methods. Lets make this method not static